src/modules/histogramcn/gui/histogramcnwidget.h

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// Copyright (c) 2023 Team Dissolve and contributors
+
+#pragma once
+
+#include "modules/hitogramcn/gui/ui_histogramcnwidget.h"
+#include "modules/widget.h"
+
+// Forward Declarations
+class HistogramCNModule;
+class DataViewer;
+
+// Module Widget
+class HistogramCNModuleWidget : public ModuleWidget
+{
+    // All Qt declarations derived from QObject must include this macro
+    Q_OBJECT
+
+    private:
+    // Associated Module
+    HistogramCNModule *module_;
+
+    public:
+    HistogramCNModuleWidget(QWidget *parent, HistogramCNModule *module, Dissolve &dissolve);
+
+    /*
+     * UI
+     */
+    private:
+    // Main form declaration
+    Ui::HistogramCNModuleWidget ui_;
+    // DataViewer contained within this widget
+    DataViewer *rdfGraph_;
```

Comment:
Needs renaming to something like:
```suggestion
    DataViewer *cnGraph_;
```